### DISCUSSION TITLE
How to remove scrollbar in QscrollArea

### DISCUSSION THREAD

**Discussion by bobwatcherx** [quasarframework/quasar#15033] [General - Components / Directives / etc] [answered]
how to remove scrollbar in scrollArea in quasar
this script is not work
```
::-webkit-scrollbar {
   display: none;
   overflow:hidden
}
```
thanks

**Comment by MilosPaunovic** [comment]
Why would you even use scrollArea without a scrollbar?

**Comment by suleimanas** [comment]
I hope you mean scroll areas in general :)
`class="no-scroll"`
https://quasar.dev/style/other-helper-classes#scroll-related

but if you mean QScrollArea use `content-style` and `content-active-style` **props**

**Reply by bobwatcherx** [reply]
> I hope you mean scroll areas in general :) `class="no-scroll"` https://quasar.dev/style/other-helper-classes#scroll-related
> 
> but if you mean QScrollArea use `content-style` and `content-active-style` **props**

![Screenshot_2022-12-05-16-43-04-22](https://user-images.githubusercontent.com/52072811/205605294-0d365fe1-8714-4678-b059-2d95328ef87d.png)
I mean just get rid of the scrollbar . And still scrollable. In that picture I scroll horizontally and how do I get rid of the scrollbar only

**Comment by bobwatcherx** [comment]
class=" no-scroll" 
Is not work

**Comment by suleimanas** [comment, marked as answer]
it can be styled, change the colors to transparent
https://quasar.dev/vue-components/scroll-area#styled
